### DISCUSSION TITLE
How to hide blog posts with certain tags with correct pagination

### DISCUSSION THREAD

**Discussion by mindnous** [alshedivat/al-folio#2667] [Q&A] [answered]
Hi all, I've been briefly tinkering with alfolio for the past month, and for context, I have zero Ruby/Jekyll background.

I am trying to hide alfolio post list for my future reference, to which I can (add {% if post.tags contains "sample-post" %}{% continue %}{% endif %} [ to this line](https://github.com/alshedivat/al-folio/blob/master/_pages/blog.md?plain=1#L112)). 
The problem is that the pagination/paginator still include all posts in its list. This causes it to show the pagination page "Newer page 1 | 2 | 3 | 4| Later", whereas I only have 4 recent post files with different tags.

FYI, my temporary solution in the end is to just delete all old posts, but I feel a great pity, since it's such a good raw material to reference to in case that I forgot (which is most likely happening).

Note: I also don't want to disable the pagination.

Thanks in advance.
BR, Brilian

**Comment by george-gca** [comment]
One solution would be to move these posts to another directory, like `_drafts/`. Drafts, for instance, is native to jekyll and you can run your blog with the `--drafts` flag and enable them.

**Reply by mindnous** [reply]
Hi @george-gca , thanks for the suggestion. Unfortunately though,  I get this error running with the additional --drafts argument. Have you also encountered the issue with that argument before?

```
Liquid Exception: Could not find post "2018-12-22-distill" in tag 'post_url'. Make sure the post exists and the name is correct. in /Users/brilian/Downloads/btnio/_drafts/2023-07-12-post-bibliography.md
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Could not find post "2018-12-22-distill" in tag 'post_url'. Make sure the post exists and the name is correct. 
                    ------------------------------------------------
      Jekyll 4.3.3   Please append `--trace` to the `serve` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
```

**Reply by mindnous** [reply]
![image](https://github.com/user-attachments/assets/7e5b9d65-ed89-4af4-844b-7e7930e74fa2){ width=20% }

**Reply by george-gca** [reply, marked as answer]
The error message is clear:

```
Could not find post "2018-12-22-distill" in tag 'post_url'... in /Users/brilian/Downloads/btnio/_drafts/2023-07-12-post-bibliography.md
```

Which means, the post about bibliography references the distill post using `post_url`, but `post_url` can only find posts that are in `_posts/`. You can either modify this post or delete it.

**Reply by chriskcovert** [reply]
where should the _drafts folder be created on the root of main? or inside the _posts folder?

I created _drafts under the _posts folder and put half the example posts inside and they still should up in the blog page and can still be navigated to and viewed normally...

**Reply by george-gca** [reply]
On the root of the main, like:
```
.
├── 📂 _drafts/
└── 📂 _posts/
```

For more information, check our [CUSTOMIZE.md](https://github.com/alshedivat/al-folio/blob/main/CUSTOMIZE.md#creating-new-blog-posts).
